scss/mixins/_nav.scss

```diff
@@ -0,0 +1,60 @@
+// Justified nav links
+// -------------------------
+
+@mixin nav-justified {
+  width: 100%;
+
+  .nav-item {
+    float: none;
+  }
+
+  .nav-link {
+    margin-bottom: 5px;
+    text-align: center;
+  }
+
+  .dropdown .dropdown-menu {
+    top: auto;
+    left: auto;
+  }
+
+  @include media-breakpoint-up(md) {
+    .nav-item {
+      display: table-cell;
+      width: 1%;
+    }
+    .nav-link {
+      margin-bottom: 0;
+    }
+  }
+}
+
+// Move borders to anchors instead of bottom of list
+//
+// Mixin for adding on top the shared `.nav-justified` styles for our tabs
+@mixin nav-tabs-justified {
+  border-bottom: 0;
+
+  .nav-link { // Override margin from .nav-tabs
+    margin-right: 0;
+    border-radius: $border-radius;
+  }
+
+  .nav-link.active,
```

Comment:
Make use of the `plain-hover-focus` mixin here.
See https://github.com/twbs/bootstrap/blob/86c0520613d37beb2b7f592e72806c4b3152d26e/scss/mixins/_hover.scss#L27